src/main/tv/codely/scala_intro_examples/lesson_05_ifs_for/Maker.scala

```diff
@@ -0,0 +1,9 @@
+package tv.codely.scala_intro_examples.lesson_05_ifs_for
```

Comment:
Instead of modifying the exercise base code I would suggest to do the very same as you've already done with the exercise number 6. That is, moving the solution to your own package `src/main/tv/codely/scala_intro_examples/lesson_05_ifs_for/exercise_solutions/daniel_albarral/*`

This way we would be able to merge your solution but also maintaining the base code for further users 🙂